tests/scripts/thread-cert/border_router/test_publish_meshcop_service.py

```diff
@@ -102,11 +102,19 @@ def test(self):
         self.simulator.go(10)
         self.check_meshcop_service(br1, host)
 
+        # check Meshcop-e service
+        self.start_ephemeral_key_mode(br1, True)
```

Comment:
nit: `start` seems not a good prefix as the method here support both activation and deactivation, and state check as well. update the name to be more descriptive?

```suggestion
        self.config_and_check_ephemeral_key_mode(br1, True)
```